wormchain/x/wormhole/keeper/msg_server_wasmd.go

```diff
@@ -12,6 +12,8 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+var WASMD_CONTRACT_ADMIN = sdk.AccAddress("wormchain")
```

Comment:
let's make this "wormchain_wasmd_owner" such that there are no potential collisions in the future